lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -560,6 +562,7 @@ readPoolDbData DBLayer {..} currentEpoch = atomically $ do
         retirementCertificates
         <$> readTotalProduction
         <*> readPoolMetadata
+        <*> (Set.fromList <$> readDelistedPools)
```

Comment:
Maybe we just pass it through as a Set? There's json instances for it anyway.

Reply:
By passing it through as a `Set`, do you mean writing a set to a single row?

I think that would make it harder to inspect the table using tools like `sqlitebrowser`.

In any case, having one row per pool ID means that we can more easily write queries on the table, especially if we need to perform a join elsewhere.

Reply:
Nvm... it's fine this way